apps/src/music/views/music-play-view.module.scss

```diff
@@ -0,0 +1,53 @@
+@import 'color';
+
+.musicPlayViewContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+  background-color: $light_black;
+
+  .musicPlayViewCard {
+    display: flex;
+    flex-direction: column;
+    max-width: 350px;
+    background-color: $light_gray_900;
+    gap: .5rem;
+    padding: 2rem;
+    border-radius: 0.5rem;
+
+    img {
+      width: 56px;
+      align-self: center;
+    }
+
+    h2, p, a {
```

Comment:
Can we use classnames for these?

Reply:
We can, but why? With current structure there's no need for className, why add them? In the future if structure becomes more complex - then I would add a classNames to elements where needed.

That being said, one thing I would do is wrap up this img, h2, p elements (a is removed now) in one div with smth like `.musicPlayViewInfoSection` for better structure

Reply:
Talked offline with @sanchitmalhotra126 and @levadadenys and we're good with this update!

Reply:
I personally prefer to be as specific as possible with CSS rules - for example if another `img` is added to this component that needs a different style, we'd have to rework the styles here instead of just adding a new class name. For this case, it's not a huge deal, but my personal preference is to use classnames where possible.

Reply:
Oh, gotcha now! Yep, agree, makes sense for the img. Thanks for explaining!

Reply:
If okay with you @levadadenys I can update in follow-up PR.